app/models/tagging.rb

```diff
@@ -0,0 +1,6 @@
+class Tagging < ApplicationRecord
+  belongs_to :taggable, :polymorphic => true
+  belongs_to :tagger,   :polymorphic => true
```

Comment:
@agrare do we need the tagger? Is this for some future use?

Reply:
It was in the table so I added an assoc for it.  Since we prob won't have our users in our db we can probably drop it from the table if we get rid of acts-as-taggable-on.